ris.go

```diff
@@ -80,34 +79,120 @@ func (f *RISFinder) Collector(name string) (Collector, error) {
 	return Collector{}, nil
 }
 
-func (f *RISFinder) Find(query Query) ([]File, error) {
-	// TODO
-	return nil, nil
+// Find the BGP data corresponding to the query
+// The naming scheme for BGP data is as follows:
+// https://data.ris.ripe.net/rrcXX/YYYY.MM/TYPE.YYYYMMDD.HHmm.gz
+func (f *RISFinder) Find(query Query) ([]BGPDump, error) {
+	var results []BGPDump
+	var allowedPrefixes []string
+
+	if query.DumpType == DumpTypeRib || query.DumpType == DumpTypeAny {
+		allowedPrefixes = append(allowedPrefixes, "bview.")
+	}
+	if query.DumpType == DumpTypeUpdates || query.DumpType == DumpTypeAny {
+		allowedPrefixes = append(allowedPrefixes, "updates.")
+	}
+
+	for _, collector := range query.Collectors {
+		// baseURL: https://data.ris.ripe.net/rrcXX
+		baseUrl := "https://data.ris.ripe.net/" + collector.Name
+		fmt.Println("Scraping ", baseUrl)
```

Comment:
This is fine for now, but let's start using a real logger -- I don't remember if I already wired on up in here, but my preference is for [zerolog](https://github.com/rs/zerolog).